holmes/plugins/toolsets/newrelic/newrelic.py

```
@@ -0,0 +1,242 @@
+        resp = PromResponse.from_newrelic_records(
+            records=records,
+            tool_name=self.name,
+            params=params or {},
+            begin_key=begin_key,
+            end_key=end_key,
+            facet_key=facet_key,
+        )
+        return resp.to_json()
+
+    def _invoke(
+        self, params: dict, user_approved: bool = False
```

Ruff (0.12.2)

Unused method argument: `user_approved`

(ARG002)